### DISCUSSION TITLE
When is `go.opentelemetry.io/collector/model` 1.0 release?

### DISCUSSION THREAD

**Discussion by ItsLastDay** [open-telemetry/opentelemetry-collector#5783] [Q&A] [answered]
Hi OTel community,

I'm interested when `pdata` package will become stable. I see that https://github.com/open-telemetry/opentelemetry-collector/milestone/24 is marked as "100% complete", and at the same time "Past due by 4 months". 

How can I track this? In other words, where should I look to get an estimate "pdata will be stable on date X"?

Thanks,
Mike

**Comment by bogdandrutu** [comment, marked as answer]
Last item was closed by mistake, still working on that.

**Reply by ItsLastDay** [reply]
Thanks! And when do you think, roughly, the last item will be completed? A month? Two months?
